src/ir_Hitachi.h

```diff
@@ -41,6 +41,15 @@ const uint8_t kHitachiAcPowerOffset = 0;
 const uint8_t kHitachiAcSwingOffset = 7;
 
 // HitachiAc424
+// Byte[11]
+const uint8_t kHitachiAc424ButtonByte = 11;
+const uint8_t kHitachiAc424ButtonPower = 0x13;
+const uint8_t kHitachiAc424ButtonMode = 0x13;
+const uint8_t kHitachiAc424ButtonFan = 0x42;
+const uint8_t kHitachiAc424ButtonTempDown = 0x43;
+const uint8_t kHitachiAc424ButtonTempUp = 0x44;
```

Comment:
Assuming these are also set when the button is pressed. Each appropriate `setFoo()` should contain an appropriate `setButton()` call. Please also update the unit tests too if you can.